index.template.html

```diff
@@ -4025,12 +4039,20 @@ <h3>Protocol Bindings</h3>
     form instance (see next section, <a href="#http-binding-assertions"></a>).
     <ul>
         <li>
-            <span class="rfc2119-assertion" id="bindings-requirements-scheme">
+            <span class="rfc2119-assertion" id="td-uri-scheme-requirement">
                 Every form in a WoT Thing Description MUST follow the requirements
                 of the <a>Protocol Binding</a> indicated by the URI scheme [[!RFC3986]] of its
-                <code>href</code> member as indicated by the Binding Templates specification at <a href="https://w3c.github.io/wot-binding-templates/#creating-a-new-protocol-binding">Creating a New Protocol Binding</a>.
-
+                <code>href</code> member.
+            </span>
+            <span class="rfc2119-assertion" id="td-possible-uri-schemes"></span>
+                The URI scheme in <code>href</code> member MUST be either registered with IANA at [[IANA-URI-SCHEMES]]
+                or associated with one of the following protocols: [[MQTT]], [[MODBUS]], [[OPC-UA]].
```

Comment:
I would not do a limitation of protocols in the TD spec. This may result to the impression that WoT TD would only work with those + listed IANA. 
I would change the sentence to  "... or associated with protocols such as for [[MQTT]], ... "